### ISSUE TITLE
Don't know how to build task 'test' (RuntimeError)

### ISSUE THREAD

**Issue by yossi-shasho** [burke/zeus#213]
After the last update, when running `zeus rspec` I started getting this exception
```
.rvm/rubies/ruby-1.9.3-p194/lib/ruby/1.9.1/rake/task_manager.rb:49:in `[]': Don't know how to build task 'test' (RuntimeError)
	from /home/joe/.rvm/rubies/ruby-1.9.3-p194/lib/ruby/1.9.1/rake/task.rb:298:in `[]'
	from /home/joe/.rvm/gems/ruby-1.9.3-p194@silo/gems/zeus-0.13.1/lib/zeus/m.rb:131:in `parse'
	from /home/joe/.rvm/gems/ruby-1.9.3-p194@silo/gems/zeus-0.13.1/lib/zeus/m.rb:120:in `run'
	from /home/joe/.rvm/gems/ruby-1.9.3-p194@silo/gems/zeus-0.13.1/lib/zeus/m.rb:106:in `run'
```
and when running guard, this exception:
```
.rvm/gems/ruby-1.9.3-p194@silo/gems/zeus-0.13.1/lib/zeus/m.rb:188:in `block in parse_options!': invalid option: -f (OptionParser::InvalidOption)
	from /home/joe/.rvm/gems/ruby-1.9.3-p194@silo/gems/zeus-0.13.1/lib/zeus/m.rb:162:in `new'
	from /home/joe/.rvm/gems/ruby-1.9.3-p194@silo/gems/zeus-0.13.1/lib/zeus/m.rb:162:in `parse_options!'
	from /home/joe/.rvm/gems/ruby-1.9.3-p194@silo/gems/zeus-0.13.1/lib/zeus/m.rb:134:in `parse'
	from /home/joe/.rvm/gems/ruby-1.9.3-p194@silo/gems/zeus-0.13.1/lib/zeus/m.rb:120:in `run'
	from /home/joe/.rvm/gems/ruby-1.9.3-p194@silo/gems/zeus-0.13.1/lib/zeus/m.rb:106:in `run'

```
Any idea anyone?

**Comment by sshao** [comment]
This should be fixed by burke/zeus#497 now

**Comment by latortuga** [comment]
This will go out in the next release, thanks @sshao
